### ISSUE TITLE
'az disk grant-access --access-level Write' needs an example

### ISSUE THREAD

**Issue by colemickens** [Azure/azure-cli#10209]
Can this please be documented? 

(Sorry, this is possibly a dupe of my other open issue, I thought this would land on a Docs repo... Either way.)

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 74997eeb-6fa3-4318-79da-8cf4dce59cfe
* Version Independent ID: c8cea09f-196e-7269-dc42-b6b8933dbaf1
* Content: [az disk](https://docs.microsoft.com/en-us/cli/azure/disk?view=azure-cli-latest#feedback)
* Content Source: [latest/docs-ref-autogen/disk.yml](https://github.com/MicrosoftDocs/azure-docs-cli/blob/live/latest/docs-ref-autogen/disk.yml)
* Service: **storage**
* GitHub Login: @rloutlaw
* Microsoft Alias: **routlaw**

**Comment by colemickens** [comment]
For example: get the SAS url and then upload a VHD to it, or do a blob copy from a blob somewhere else in some Azure storage account.

**Comment by yonzhan** [comment]
add to S163.

**Comment by 08nholloway** [comment]
@colemickens we have a few end to end examples that are similar to what you described:

https://github.com/Azure-Samples/azure-cli-samples/blob/master/virtual-machine/copy-managed-disks-vhd-to-storage-account/copy-managed-disks-vhd-to-storage-account.sh

https://github.com/Azure-Samples/azure-cli-samples/blob/master/virtual-machine/copy-snapshots-to-storage-account/copy-snapshots-to-storage-account.sh

Neither of them specifically use the --access-level parameter though, so if you'd like an example with I can write one up.

**Comment by colemickens** [comment]
We can close this, I don't need an example. It now works as one might expect. From my recollection, when I filed this the service wasn't working very... predictably.

Also, when I wrote this up and tested it, due to Azure's polling, this method of direct pushing to a managed disk blob was actually slower than pushing to storage and then creating a MD from the blob. Which of course means that I have to still name and manage storage accounts, but at least I'm not just waiting non-trivial amounts of time to just take/release a lease.
